architecture/core_concepts/routes.adoc

```diff
@@ -802,3 +802,34 @@ If unit not provided, ms is the default.
 Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
+
+
+[[wildcard-subdomain-route-policy]]
+== Creating routes specifying a wildcard subdomain policy
```

Comment:
put headings in title case:
Creating Routes Specifying a Wildcard Subdomain Policy